documentation/GuideLines.md

```diff
@@ -0,0 +1,165 @@
+# DRAFT
+This is a draft of the proposed code standards and best practices for the Mission Control Project going forward to help facilitate a productive cross-team workflow to a unified standard. The plan is to use this standard to outline the workflow milestones for the more long term restructuring of some components needed to fulfill the vision of Mission Control envisioned by the stakeholder.
+
+# GitFlow - Git pull requests practices and standards
+### Create a branch
+* Pull down from Master to your local machine. This will assure that you have the latest and greatest from the repo!
+```
+git pull master
+```
+* Create/checkout a local feature branch. Once you have the master locally, create a branch of your feature.
+```
+git checkout -b feature/"YOUR-FEATURE-NAME"
+```
+
+### Work Flow
+* Be mindful of GitHub updates and announcements posted to the Slack channel.
+* Strive to keep individual contributions under 500 changed lines. 
+* Update Draft Pull Request description to include a detailed description of the new feature or bug fix that being is added.
+* Provide relevant in code comments to clarify changes when needed 
+* Update appropriate ```Readme.md```
+* Update QA Feature Checklist to reflect changes
+* Update master Trello for PRs
+ ### Things to do before taking pull request out of draft:
+* Remove any commented out code.
+* Remove any console logs.
+* Pull master into your branch and resolve git conflicts
+```
+git pull origin master
+```
+* Let your TL know you are taking the pull out of draft.
+
+### Pull Reqest Review
+* 2 reviews are required to merge code to master. (One revew must be by a TL)
+* Any code, regardless of functionality, that fails: QA, or TL review, or style rules will be rejected, to allow the needed changes to be made.
+
+# 
+# QA Flow
+* Maintain a list of current app features so a reviewer knows what functionality to check for.
+### Review
+* Download the branch and test localy that all features work. Remember to update the Prisma data and seed:
+```
+npm run reseed
+```
+will run
+```
+prisma delete
+prisma generate
+prisma deploy
+```
+* If avalible, test on staging enviroment.
+* Check all current and new features work fully prior to merging to master.
+* Per Rejected Pull Request & Feedback Flow leave aproprate coments.
+
+# 
+# Rejected Pull Request & Feedback Flow
+* GitHub comments should be code-centric and professional at all times.
```

Comment:
it would be good to provide a good example or link to a great comment

Reply:
Based on this feedback added 4 ```Scenario:``` bad/good comment sets here to provide examples.